src/luarocks/fs/win32/tools.lua

```diff
@@ -267,7 +267,7 @@ function tools.set_permissions(filename, mode, scope)
          return false, "Failed setting permission " .. mode .. " for " .. scope
       end
       -- Grant permissions available only to the current user
-      ok = fs.execute_quiet(vars.ICACLS .. " " .. fs.Q(filename) .. " /inheritance:d /grant %USERNAME%:" .. my_perms)
+      ok = fs.execute_quiet(vars.ICACLS .. " " .. fs.Q(filename) .. " /inheritance:d /grant \"%USERNAME%\"":" .. my_perms)
```

Comment:
@CannibalVox There is a typo here (`\""`), that's what causing the build failures.

```suggestion
      ok = fs.execute_quiet(vars.ICACLS .. " " .. fs.Q(filename) .. " /inheritance:d /grant \"%USERNAME%\":" .. my_perms)
```

I fixed it and I'm rerunning the Windows CI. If this passes, I'll merge your commit manually! 

https://ci.appveyor.com/project/hishamhm/luarocks/builds/29214565